src/main/java/org/opentripplanner/routing/graph/GraphIndex.java

```diff
@@ -78,6 +78,9 @@
 
     /** maximum distance to walk after leaving transit in Analyst */
     public static final int MAX_WALK_METERS = 3500;
+    
+    /** Should stop clusters be created by parent stations instead of by proximity?  */
+    public static final boolean CLUSTER_BY_PARENT_STATIONS = false;
```

Comment:
This seems sort of funny - it's a final variable with a fixed value, so all of the other code you're adding is dead.  Do you mean to make this configurable?